indico/migrations/versions/20230810_1300_0af8f63aa603_refactor_editing_revisions.py

```diff
@@ -240,7 +249,13 @@ def upgrade():
             INSERT INTO event_editing.revisions
             (editable_id, user_id, created_dt, type, is_undone, comment) VALUES
             (%s,          %s,      %s,         %s,   %s,        %s)
-        ''', rev)
+        ''', rev[:-1])
+        for comment in rev[-1]:
+            conn.execute('''
+                UPDATE event_editing.comments
+                SET revision_id = (SELECT id FROM event_editing.revisions ORDER BY id DESC LIMIT 1)
```

Comment:
That looks incredibly fragile considering the subquery has no correlation with the revision/editable whatsoever...